Business/Services/Query/DocumentQueryService.cs

```diff
@@ -29,13 +29,14 @@ public DocumentQuery<TDocument> GetDocuments<TDocument>() where TDocument : Tree
             if (SiteContext.IsPreviewEnabled)
             {
                 query = query
-                    .Columns("NodeSiteId")
+                    .Columns("NodeSiteId") // Set initial columns returned for optimization.  If not set, all columns are returned.
                     .OnSite(SiteContext.SiteName) // There could be more sites with matching documents
                     .LatestVersion()
                     .Published(false)
                     .Culture(SiteContext.PreviewCulture);
             } else {
                 query = query
+                    .Columns("NodeSiteId") // Set initial columns returned for optimization.  If not set, all columns are returned.
```

Comment:
@eugene-cp 'NodeSiteId' is not required here, is it? If so, could we just pass an empty array? Or would that produce `select * from` query? If it did, I would probably just added `DocumentId` column instead as this gets loaded anyway (even if its not specified)

Reply:
@Enngage  Yes, in Live Mode, you don't need "NodeSiteId", however, passing an empty array will generate a `select *` query.  Revised the default column to "DocumentId" as recommended.

Please see query below in Live Mode:
![image](https://user-images.githubusercontent.com/22228592/51544584-e10f2480-1e25-11e9-97c2-5d73b909f6e8.png)